include/cpr/filesystem.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CPR_FILESYSTEM_H
+#define CPR_FILESYSTEM_H
+
+// Include filesystem into the namespace "fs" from either "filesystem" or "experimental/filesystem"
+#if __has_include(<filesystem>)
+#include <filesystem>
+namespace fs = std::filesystem;
+#elif __has_include("experimental/filesystem")
+#include <experimental/filesystem>
+namespace fs = std::experimental::filesystem;
+#else
+#error "Failed to include <filesystem> header!"
+#endif
+
+#endif
```

Comment:
I would prefer to not have this header. But I can't think of any neater solution, … so it is fine

Reply:
Yeah, me neither. But I think it is the best solution for now.